### PR TITLE
Add FerretDB to marketplace

### PR DESCRIPTION
**Pull request by b1ron**
Closes FerretDB/engineering#74.

### CODE REVIEW

File: FerretDB/assets/95204649.jpeg
Comment (AlekSi) [inline FerretDB/assets/95204649.jpeg:1]: Let's rename the file to something better

File: FerretDB/ferretdb_stackscript.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# <UDF name="username" label="Username for authentication" default="username" />
+# <UDF name="password" label="Password for authentication" default="password" />
+# <UDF name="ipaddr" label="Listen TCP address" default="127.0.0.1" />
+
```
Comment (AlekSi) [inline FerretDB/ferretdb_stackscript.sh:6]: `export VERSION=1.6.0` to update in one place later

File: FerretDB/ferretdb_stackscript.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+apt update && apt upgrade -y && apt install -y wget postgresql postgresql-contrib
+
+wget https://github.com/FerretDB/FerretDB/releases/download/v1.6.0/ferretdb.deb
```
Comment (AlekSi) [inline FerretDB/ferretdb_stackscript.sh]: Use `VERSION` there

File: FerretDB/ferretdb_stackscript.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+sudo -u postgres createdb ferretdb --locale=en_US.UTF-8 --template=template0
+sudo -u postgres psql -c "GRANT CREATE ON DATABASE ferretdb TO $USERNAME;"
+
+cat <<END >/lib/systemd/system/ferretdb.service
+[Unit]
+Description=FerretDB is an open-source proxy, converting the MongoDB 6.0+ wire protocol queries to SQL - using PostgreSQL or SQLite as a database engine.
+# https://docs.ferretdb.io
+After=network.target
+
+[Service]
```
Comment (AlekSi) [inline FerretDB/ferretdb_stackscript.sh:29]: Let's add `FERRETDB_TEST_TELEMETRY_PACKAGE=linode` environment variable there

File: FerretDB/ferretdb_stackscript.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+Type=simple
+ExecStart=/usr/bin/ferretdb --listen-addr=$IPADDR:27017 --debug-addr="127.0.0.1:8089"
```
Comment (AlekSi) [inline FerretDB/ferretdb_stackscript.sh:32]: Alternatively, we could use `--test-telemetry-package=linode`

File: FerretDB/assets/ferretdb_logo.jpeg
Comment (AlekSi) [inline FerretDB/assets/ferretdb_logo.jpeg:1]: Let's replace it with SVG

Comment (b1ron) [reply FerretDB/assets/ferretdb_logo.jpeg:1]: Done

**Pull request merged**